.gitignore

```diff
@@ -1,27 +1,13 @@
 /app/config/parameters.yml
-/app/data/images/*
-!/app/data/images
-!/app/data/images/.gitkeep
+/app/data/images/
```

Comment:
It's just because using ! sign have no effect on the files already added in the Git index. I prefer to ignore an entire folder and manually add the .gitkeep file in the index (ie: added by force, regardless of the ignore list).

By this way, the folder and it's contents will always be ignored AND the folder will be created thanks to the .gitkeep file AND there is no useless line in the gitignore.